backend/prisma/schema.prisma

```diff
@@ -67,3 +67,9 @@ model Calendar {
 
   workshop Workshop @relation(fields: [workshopId], references: [id])
 }
+
+model Subscription {
```

Comment:
Klopt het dat hier alleen wat word ingezet en nooit wat uitgehaald? Waarvoor is deze nodig?